packages/flutter/lib/src/cupertino/localizations.dart

```diff
@@ -82,12 +82,17 @@ abstract class CupertinoLocalizations {
   /// Day of month that is shown in [CupertinoDatePicker] spinner corresponding
   /// to the given day index.
   ///
+  /// if weekDay is provided then it will also show Day of week alongside day
```

Comment:
nit: format to "If weekDay is provided then it will also show weekday name alongside the numerical day."

Reply:
Done.